File: space-game/OrbitalMechanics/Classes/RingSystem.vb
Imports Core.Classes

Namespace Classes
    Public Class RingSystem
        Public Property InnerRadius() As Distance
        Public Property OuterRadius() As Distance
        Public Property Textures() As List(Of Texture)

        Public Sub New(innerRadius As Distance, outerRadius As Distance, textures As List(Of Texture))
            Me.InnerRadius = innerRadius
            Me.OuterRadius = outerRadius
            Me.Textures = textures
        End Sub
    End Class
End NameSpace

File: space-game/OrbitalMechanics/Classes/RingSystem.cs
using Core.Classes;
using System.Collections.Generic;

namespace OrbitalMechanics.Classes
{
    public class RingSystem
    {
        public Distance InnerRadius { get; set; }
        public Distance OuterRadius { get; set; }
        public List<Texture> Textures { get; set; }

        public RingSystem(Distance innerRadius, Distance outerRadius, List<Texture> textures)
        {
            InnerRadius = innerRadius;
            OuterRadius = outerRadius;
            Textures = textures;
        }
    }
}
